lib/security/password.go

```diff
@@ -1,13 +1,14 @@
 package security
 
 import (
-	"golang.org/x/crypto/bcrypt"
+	"crypto/sha256"
+	"encoding/hex"
 )
 
 // HashPassword : Hash Password
 func HashPassword(password string) string {
-	bytes, _ := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
-	password = string(bytes)
+	bytes := sha256.Sum256([]byte(password))
+	password = hex.EncodeToString(bytes[:])
```

Comment:
This weakens the security, since one can use rainbow-tables or simple Google search to decode the password. For example `5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8` is easily to be revealed as `password` via Google search.

At least use HMAC-SHA256 with constant hash, which is harder to be reverted. Something like:

```
secret := "LndHub.go is the best"
h := hmac.New(sha256.New, []byte(secret))
h.Write([]byte(password))
password = hex.EncodeToString(h.Sum(nil))
```

Reply:
Yeah, makes sense

Reply:
One of my teammates raised some concerns about breaking the de-facto standard for storing passwords in databases. I think I can revert it to `bcrypt` and handle the password,login collisions on code instead of doing it at the db level. I am doing it anyway for enforcing login/nickname uniqueness, so I don't think is a step back and at least we don't weaken the security we previously had.